### PR TITLE
Adding pypi and travis ci badges  #6825

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+nereid-review
+=============
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-review.svg?branch=3.4.0.1)](https://travis-ci.org/openlabs/nereid-review)
```
Comment (aroraumang) [inline README.md]: This should be for develop branch.

**Comment by PritishC** [comment]
LGTM.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+nereid-review
+=============
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-review.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-review)
+[![Downloads](https://pypip.in/download/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_trytond_nereid_review/)
+[![Latest Version](https://pypip.in/version/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_review/)
+[![Development Status](https://pypip.in/status/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_review/)
+
```
Comment (PritishC) [inline README.md]: Please add the coveralls badge.

Comment (PritishC) [inline README.md]: There is no paragraph after this blank line, so there is no need for it.

Please remove it.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+nereid-review
+=============
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-review.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-review)
+[![Downloads](https://pypip.in/download/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_trytond_nereid_review/)
+[![Latest Version](https://pypip.in/version/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_review/)
+[![Development Status](https://pypip.in/status/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_review/)
```
Comment (aroraumang) [inline README.md:7]: Coveralls badge?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+nereid-review
+=============
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-review.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-review)
+[![Downloads](https://pypip.in/download/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_trytond_nereid_review/)
```
Comment (PritishC) [inline README.md]: Downloads link is incorrect.

**Comment by PritishC** [comment]
LGTM.

**Comment by prakashpp** [comment]
@rituparnapanda please merge

**Pull request merged**